docs/ceph-csi-upgrade.md

```diff
@@ -48,6 +49,43 @@ fuse client or rbd-nbd as of now.
 This guide will walk you through the steps to upgrade the software in a cluster
 from v2.0 to v2.1
 
+### RBD CSI Snapshots
+
+CSI snapshot is moved from Alpha to Beta and is not backward compatible. The snapshots
+created with the Alpha version must be deleted before the upgrade.
```

Comment:
can you add a check of how users can verify that they have the alpha version of the CRD installed?